test/ectest.c

```diff
@@ -2123,9 +2123,7 @@ static int custom_generator_test(int id)
         goto err;
 
     /* expected byte length of encoded points */
-    bsize = (EC_GROUP_get_field_type(group) == NID_X9_62_prime_field) ?
-             BN_num_bytes(EC_GROUP_get0_field(group)) :
-             (EC_GROUP_get_degree(group) + 7) / 8;
+    bsize = (EC_GROUP_get_degree(group) + 7) / 8;
```

Comment:
@bbbrumley Am I missing why (other than for clarity) you did not use `EC_GROUP_get_degree()` in both cases?

Reply:
Not really. It started out with no `EC_GROUP_get_degree` or branches at all but then failed for some old binary curves. So then I went digging for how to make that work and didn't back up to reconsider. (I would not have expected `EC_GROUP_get_degree` to do anything sane for GFp curves, but looks like I'm wrong.)